docs/obs/traffic-inspection.mdx

```diff
@@ -25,14 +19,11 @@ ngrok offers multiple ways to inspect the traffic going to your service.
 
 ## ngrok Traffic Inspector
 
-:::note
+The ngrok Traffic Inspector replaces the local agent inspection interface and adds additional capabilities such as longer retention and searching.
+By enabling full capture mode in your ngrok settings, you are able to replay requests to your upstream directly from the browser, instead of the original source.
+This can dramatically speed up development of webhooks or callbacks that are difficult to trigger repeatedly.
 
-ngrok Traffic Inspector is currently in developer preview and only available to
-specific customers. While in developer preview, you will not be charged for its usage, things will change and break
-without notice, and data may be lost at any time. If you have feedback or questions
-please [reach out to the product team](mailto:support@ngrok.com?subject=Traffic%20Insights%20Feedback).
-
-:::
+Everyone gets three days of traffic retention after which, the traffic events are purged from the system. If you would like a longer look back window, you can purchase up to [90 days of retention with our production plans](https://ngrok.com/pricing).
```

Comment:
`Everyone gets three days of traffic retention after which, the traffic events are purged from the system.`

perhaps should be

`Everyone gets three days of traffic retention, after which the traffic events are purged from the system.`

Reply:
```suggestion
Everyone gets three days of traffic retention, after which the traffic events are purged from the system. If you would like a longer look back window, you can purchase up to [90 days of retention with our production plans](https://ngrok.com/pricing).
```